src/sentry/plugins/__init__.py

```diff
@@ -1,7 +1,3 @@
 from __future__ import absolute_import
 
 HIDDEN_PLUGINS = ("bitbucket", "gitlab", "github", "slack", "vsts", "jira", "jira_ac")
-
-from sentry.plugins.base import *  # NOQA
```

Comment:
You could not delete this for the moment and then be able to merge this to master before you finish auditing getsentry/sentry-plugins/the other junk

Reply:
You're right, good call.

Reply:
The other stuff turned out to not be too bad at all, so never mind.